patterns/hero-product-split.php

```diff
@@ -6,17 +6,20 @@
  */
 ?>
 
-<!-- wp:media-text {"align":"full","mediaPosition":"right","mediaType":"image","mediaSizeSlug":"full","imageFill":false,"backgroundColor":"contrast","textColor":"base"} -->
-<div class="wp-block-media-text alignfull has-media-on-the-right is-stacked-on-mobile has-base-color has-contrast-background-color has-text-color has-background">
-	<div class="wp-block-media-text__content"><!-- wp:heading -->
-		<h2 class="wp-block-heading">Get cozy this fall with flannel shirts</h2>
+<!-- wp:media-text {"align":"full","mediaPosition":"right","mediaType":"image","mediaSizeSlug":"full","imageFill":false,"style":{"color":{"background":"#000000","text":"#ffffff"}}} -->
+<div class="wp-block-media-text alignfull has-media-on-the-right is-stacked-on-mobile has-text-color has-background" style="color:#ffffff;background-color:#000000">
+	<div class="wp-block-media-text__content">
+		<!-- wp:heading {"textColor":"background"} -->
```

Comment:
I'm not sure to understand this part `{"textColor":"background"}`. This is how it looks in Storefront by default and when I change the heading color to white (my understanding is that the desired state is the second one?):

This PR | Changing the heading color to white
--- | ---
![imatge](https://github.com/woocommerce/woocommerce-blocks/assets/3616980/09c28286-589d-49fb-b7fc-578231b6aa68) | ![imatge](https://github.com/woocommerce/woocommerce-blocks/assets/3616980/4ef6db53-49c8-485e-bd97-771bc18a93df)

Reply:
Nice catch! That should actually have been changed to the hex value for white but was missed.

Updated in 287adb4.